server/src/metadata/relation-metadata/relation-metadata.service.ts

```diff
@@ -168,16 +168,16 @@ export class RelationMetadataService extends TypeOrmQueryService<RelationMetadat
 
     const createdFieldMap = createdFields.reduce((acc, fieldMetadata) => {
       if (fieldMetadata.type === FieldMetadataType.RELATION) {
-        acc[fieldMetadata.objectMetadataId] = fieldMetadata;
+        acc[fieldMetadata.name] = fieldMetadata;
       }
 
       return acc;
     }, {});
 
     const createdRelationMetadata = await super.createOne({
       ...record,
```

Comment:
I'm surprise about the usage of "record" here!?
it should be relation or relationMetadata

Reply:
Doesn't really have preference on that as it's referencing the user `input`, in fact it should be something like `relationInput` but name `record` is used in other places in the code for inputs.
I can take a quick look and rename them ? Or maybe we can create a small ticket for that ?

Reply:
let's create a small ticket for that! But yes record should be only used to describe the data itself.
The full naming on the project is: an object **record** is an instance of an object **metadata** (class)